pkg/virt-launcher/virtwrap/agent-poller/agent_poller_test.go

```diff
@@ -96,22 +96,23 @@ var _ = Describe("Qemu agent poller", func() {
 		})
 
 		It("executes the agent commands based on the time interval specified", func() {
-			const interval = 1
-			const expectedExecutions = 3
+			// interval must be bigger than initial interval (1sec), so the exponential backoff will kick in.
+			const interval = 2
+			const expectedExecutions = 4
 
 			commandExecutions := runPollAndCountCommandExecution(interval, expectedExecutions, 0)
 
 			Expect(commandExecutions).To(Equal(expectedExecutions))
 		})
 
 		It("executes the agent commands based on the minimum interval at initial run", func() {
-			const interval = 30
-			const expectedExecutions = 2
+			const interval = 3
+			const expectedExecutions = 3
 
-			// Given the initial interval is 10sec, the code under test is expected to execute the commands at time:
-			// 0, 10sec, 10sec + 2*10sec
-			// Therefore, setting a timeout limit of 20sec should cover the first 2 executions.
-			t := 2 * pollInitialInterval
+			// Given the initial interval is 1sec, the code under test is expected to execute the commands at time:
+			// 0, 1sec, 1sec + 2*1sec
+			// Therefore, setting a timeout limit of 4sec should cover the first 3 executions.
+			t := 4 * time.Second
```

Comment:
It will be safer to use 10sec, go-routines may take some time until they start.
This time is only relevant in case of failures, if everything works correctly, it should pass quickly.

Reply:
We had 1 second extra, i think it safe, but anyhow, done